### PR TITLE
ai slop

### PR DESCRIPTION
**Pull request by robobun**
This PR has been marked as AI slop and the description has been updated to avoid confusion or misleading reviewers.

Many AI PRs are fine, but sometimes they submit a PR too early, fail to test if the problem is real, fail to reproduce the problem, or fail to test that the problem is fixed. If you think this PR is not AI slop, please leave a comment.

### CODE REVIEW

File: src/md/parser.rs [hunk 1]
```diff
@@ -335,14 +342,13 @@ pub fn render_to_html(
...
-    let mut parser = Parser::init(input, flags, html_renderer.renderer());
+    let mut parser = Parser::init(input, flags, html_renderer.renderer())?;
 
     // HtmlRenderer never returns JSError/JSTerminated, so OutOfMemory is the only possible error.
     match parser.process_doc() {
         Ok(()) => {}
-        Err(ParserError::OutOfMemory) => return Err(ParserError::OutOfMemory),
         Err(ParserError::JSError) | Err(ParserError::JSTerminated) => unreachable!(),
-        Err(ParserError::StackOverflow) => return Err(ParserError::StackOverflow),
+        Err(e) => return Err(e),
```
[the remark this answers is not in the document]
Comment (robobun) [reply src/md/parser.rs]: Updated in a41de9d8e4.

File: src/runtime/cli/run_command.rs [hunk 2]
```diff
@@ -3562,6 +3562,15 @@ impl RunCommand {
...
             }
+            Err(bun_md::parser::ParserError::InputTooLarge) => {
+                pretty_errorln!(
+                    "<r><red>error<r>: markdown input is too large ({} bytes); maximum is {} bytes",
+                    contents.len(),
+                    bun_md::parser::MAX_INPUT_SIZE,
+                );
+                Output::flush();
+                Global::exit(1);
+            }
```
[the remark this answers is not in the document]
Comment (robobun) [reply src/runtime/cli/run_command.rs:3573]: Skipping this one: exercising the CLI branch needs a >4 GiB `.md` file on disk, which is too slow and disk-hungry for CI. The guard itself lives in `Parser::init`, shared by both the JS API and the CLI path, and is covered end-to-end by the Buffer-based test in `md-edge-cases.test.ts`. The CLI arm only differs in error formatting and `exit(1)`, which mirrors the adjacent `StackOverflow` arm.

File: src/runtime/api/MarkdownObject.rs [hunk 3]
```diff
@@ -137,6 +146,9 @@ pub fn render_to_ansi(global_this: &JSGlobalObject, callframe: &CallFrame) -> Js
         }
         Err(ParserError::OutOfMemory) => return Err(global_this.throw_out_of_memory()),
         Err(ParserError::StackOverflow) => return Err(global_this.throw_stack_overflow()),
+        Err(ParserError::InputTooLarge) => {
+            return Err(throw_input_too_large(global_this, input.len()));
+        }
```
[the remark this answers is not in the document]
Comment (robobun) [reply src/runtime/api/MarkdownObject.rs:151]: Good catch. Added an early size check at the top of `render_to_ansi` (0fc71e1340) so oversized input is rejected before `AnsiRenderer::init` does its 1.5×len reserve. Covers both `Bun.markdown.ansi` and the CLI `.md` path, which share this entry point. `HtmlRenderer::init` and `ImageUrlCollector::init` do no input-proportional allocation, so the other paths were already ordered correctly.

File: src/md/parser.rs [hunk 4]
```diff
@@ -135,12 +135,17 @@ impl Default for BlockHeader {
...
+/// `OFF` (= `u32`), so anything larger cannot be addressed.
+pub const MAX_INPUT_SIZE: usize = OFF::MAX as usize;
+
 #[derive(Debug, Clone, Copy, PartialEq, Eq, strum::IntoStaticStr)]
 pub enum ParserError {
     OutOfMemory,
     JSError,
     JSTerminated,
     StackOverflow,
+    InputTooLarge,
```
[the remark this answers is not in the document]
Comment (robobun) [reply src/md/parser.rs:148]: Updated in 0fc71e1340.

File: scripts/verify-baseline-static/allowlist-x64.txt [hunk 5]
```diff
@@ -966,3 +966,18 @@ _ZN9bun_image9N_AVX3_DLL18NearestPaletteImplEPKhjiiii  [AVX]
...
+#
+# ENABLE(LLINT_EMBEDDED_OPCODE_ID) is on for x86_64, so each LLInt opcode
+# handler starts with a raw 4-byte ".int <opcode_id>" (LowLevelInterpreter.cpp
+# EMBED_OPCODE_ID_IF_NEEDED). The linear-sweep decoder desyncs on those
+# embedded ints and, depending on how preceding .text aligns, can decode a
+# stray VEX-encoded instruction out of the noise. offlineasm's x86 backend
+# emits SSE2 only; the emulator phase exercises the LLInt under Nehalem and
+# passes. Same reasoning and ceiling as allowlist-x64-windows.txt llint_entry.
+# (1 symbol)
+# ----------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (robobun) [reply scripts/verify-baseline-static/allowlist-x64.txt:982]: This entry is a data-in-.text false positive, not a runtime-dispatched symbol, so there is no CPUID/HWCAP gate to name. The header says "Data-in-.text false positive, NOT a gate" for exactly that reason, matching the identical `llint_entry` entry in `allowlist-x64-windows.txt:877-890`. The triage guide's `# Gate: ...` format is for gated dispatch targets; for confirmed false positives it says "Note the reason in the group comment", which this does.
